build.gradle

```diff
@@ -73,16 +74,30 @@ subprojects {
         mavenCentral()
         mavenLocal()
     }
-
-    checkstyle {
-        toolVersion = '8.12'
-        configFile = rootProject.file('config/checkstyle/checkstyle.xml')
+    
+    spotless {
+        java {
+            target project.fileTree(project.rootDir) {
+                include '**/*.java'
+                exclude '**/package-info.java'
+            }
+            importOrder 'cern', 'com', 'io', 'org', 'reactor', 'java', 'javax', 'static *'
+            eclipse().configFile 'config/checkstyle/micrometer-style.xml'
+            licenseHeaderFile rootProject.file('config/checkstyle/copyright.txt')
```

Comment:
We do have the hieronymous license plugin applied currently which is doing some work on this. Are they overlapping?